src/Explorer/Menus/CommandBar/CommandBarComponentAdapter.tsx

```diff
@@ -55,15 +56,15 @@ export const CommandBar: React.FC<Props> = ({ container }: Props) => {
   const uiFabricControlButtons = CommandBarUtil.convertButton(controlButtons, backgroundColor);
   uiFabricControlButtons.forEach((btn: ICommandBarItemProps) => (btn.iconOnly = true));
 
+  if (NotebookUtil.isPhoenixEnabled()) {
+    uiFabricControlButtons.unshift(CommandBarUtil.createConnectionStatus(container, "connectionStatus"));
+  }
+
   if (
+    useTabs.getState().activeTab?.tabKind === ViewModels.CollectionTabKind.NotebookV2 &&
     userContext.features.notebooksTemporarilyDown === false &&
-    userContext.features.phoenix === true &&
-    useTabs.getState().activeTab?.tabKind === ViewModels.CollectionTabKind.NotebookV2
+    userContext.features.phoenix === false
```

Comment:
if (PhoenixEnabled && tabkind == Notebookv2)


always put easier flag checks first since it is and operator.